manifest.yml

```diff
@@ -8,7 +8,6 @@ applications:
     CLAIMANT_SERVICE_URL: http://htbhf-claimant-service((space-suffix)).apps.internal:8080
     NODE_ENV: production
     APP_VERSION: ((app-version))
-    CACHE_VIEW_TEMPLATES: true
```

Comment:
This should be renamed to NO_CACHE_VIEW_TEMPLATES instead of being removed.

Reply:
But it will be set to false

Reply:
I see, no point of setting it then.